packages/firebase_core/android/src/main/java/io/flutter/plugins/firebase/core/FlutterFirebaseAppRegistrar.java

```diff
@@ -8,7 +8,7 @@
 
 public class FlutterFirebaseAppRegistrar implements ComponentRegistrar {
   private static final String LIBRARY_NAME = "flutter-fire-core";
-  private static final String LIBRARY_VERSION = "0.4.0+3";
+  private static final String LIBRARY_VERSION = BuildConfig.FLUTTER_FIRE_VERSION;
```

Comment:
I don't think you need to declare it here; you can just use `BuildConfig.VERSION_NAME` below.

Reply:
done